### DISCUSSION TITLE
Lesson 10: The modulo operation Errors in the maths

### DISCUSSION THREAD

**Discussion by r0bchain** [Cyfrin/foundry-full-course-cu#1937] [General]
Hi there, I know this is not very relevant (and very basic), but it could be confusing for people who are a bit insecure with math. The calculation of the modulo is incorrect in some cases
```
2 % 3 != 1 , 2 % 3 = 2
2 % 6 != 0 , 2 % 6 != 2
2%7 != 1 , 2%7 = 2
```
AI explain this better than me :) 
![Screenshot from 2024-07-03 17-39-27](https://github.com/Cyfrin/foundry-full-course-cu/assets/32865524/64327556-200d-451d-b67f-53e709a1dd9e)

**Comment by EngrPips** [comment]
Hey @r0bertinski, Thank you so much for this heads-up this will be attended to appropriately.
